frontend/src/ui/SearchableSelect.tsx

```diff
@@ -208,5 +211,24 @@ const formatVideoListOption = (videoList: VideoListOption, _: TFunction) => (
     <div>
         <div>{videoList.title}</div>
         <SmallDescription css={{ margin: 0 }} lines={1} text={videoList.description} />
+        {videoList.created && <DateWithIcon dateString={videoList.created} />}
     </div>
 );
+
+const DateWithIcon: React.FC<{ dateString: string }> = ({ dateString }) => {
+    const { i18n } = useTranslation();
+    const date = new Date(dateString);
+
+    return <div css={{
+        display: "flex",
+        gap: 4,
+        fontSize: 10,
+        color: COLORS.neutral60,
+        alignItems: "center",
```

Comment:
I feel like this `display: flex; gap: 4; align-items: center` has been written tons of times already. I wonder if it makes sense to make a component for "icon next to text"? But doesn't have to happen in this PR, just randomly noticed it.

Reply:
Not sure if that is worth it. There are only two other cases that have these exact properties as far as I can tell.

Reply:
Ah yeah ok then